sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -314,6 +345,22 @@ export interface BlobGetPropertiesOptions extends CommonOptions {
    * @memberof BlobGetPropertiesOptions
    */
   customerProvidedKey?: CpkInfo;
+  /**
+   * An opaque DateTime value that, when present, specifies the blob
+   * snapshot to retrieve.
+   * 
+   * @type {string}
+   * @memberof BlobGetPropertiesOptions
+   */
+  snapshot?: string;
+  /**
+   * An opaque DateTime value that, when present, specifies the version
+   * of the blob to retrieve. It for service version 2019-10-10 and newer.
+   * 
+   * @type {string}
+   * @memberof BlobGetPropertiesOptions
+   */
+  versionId?: string;
```

Comment:
should we validate the case when snapshot and versionId are specified in the same time?

Reply:
Sure.

Reply:
Added test and it will throw `MutuallyExclusiveQueryParameters` for now.